internal/config/secrets.go

```diff
@@ -57,6 +58,8 @@ func loadSecrets(config *load.Config) error {
 		results := map[string]interface{}{}
 
 		switch secret.Kind {
+		case "aws-ssm":
+			secretResult = awsssmFetch(name, tempSecret)
 		case "aws-kms":
 			if secret.Region == "" {
```

Comment:
is this check actually too stringent? the region is only used if config files are not provided